model/execution/DeliveryExecutionList.php

```diff
@@ -115,7 +115,13 @@ private function createExecution($cachedData, $extraFields): array
             'id' => $cachedData[DeliveryMonitoringService::DELIVERY_EXECUTION_ID],
             'delivery' => array(
                 'uri' => $cachedData[DeliveryMonitoringService::DELIVERY_ID],
-                'label' => $this->sanitizeUserInput($cachedData[DeliveryMonitoringService::DELIVERY_NAME]),
+                'label' => $this->sanitizeUserInput((
+                    array_key_exists(DeliveryMonitoringService::DELIVERY_BATTERY_NAME, $cachedData) &&
```

Comment:
I think we should delete mentions of batteries in the taoProctoring extension

Reply:
is battery extension specific for nccer?  we don't need override label by battery name in common?

Reply:
yes, taoProctoring is not linked with batteries. I think based on the second PR you can just revert those changes